### ISSUE TITLE
dashboard sensingkit: audio-level

### ISSUE THREAD

**Issue by tlodge** [me-box/databox#124]
Do not get any data for the audio-level sensingkit sensor.  Confirmed that light and accelerometer work as expected.  Using nexus 7, and have had audio-level work fine with standalone version of the sensingkit app.

**Comment by ktg** [comment]
Yeah. I know audio-level is currently not currently working. It requires a runtime permission request. Location only needs the permission when you choose to use location as a sensor. Audio-level however requires the permission as soon as the sensor kit stuff starts running regardless or whether you're using audio-level or not. I haven't quite worked out the best way to deal with that yet.

**Comment by Toshbrown** [comment]
Could we just remove it from the driver for now?

**Comment by yousefamar** [comment]
(IIRC this is because SensingKit was written before runtime permissions were a thing (when you had to agree to all permissions while installing) and ideally should be fixed upstream in SensingKit, no?)

**Comment by Toshbrown** [comment]
@ktg is this still an issue?
